### ISSUE TITLE
Front-end: Keyboard and screen reader focus flow improvements

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#174]
Please watch this video for what I think an improved user flow would be here. https://watch.screencastify.com/v/A2e3EM1Ukn1S6qU1A4De

**Comment by amberhinds** [comment]
I think this is a lot better. Here are a few more enhancements:

1. Next and Previous buttons can be more clear if labeled "Next accessibility issue" and "previous accessibility issue"
2. This should be an H2:
`<div class="edac-highlight-panel-controls-title">Accessibility Checker</div>`
3. This should also be an H2:
`<div class="edac-highlight-panel-description-title">Image Missing Alternative Text <span class="edac-highlight-panel-description-type edac-highlight-panel-description-type-error" aria-label=" Issue type: error"> error</span></div>`
4. Instead of going to the close button when the focus is shifted into the highlight panel, can we go to the heading?  A possible fix for this is to add the close button as the last element in the highlight panel instead of the first and then to absolute position it.

**Comment by amberhinds** [comment]
I'll plan some time to have Ragavendra test this after it's released to get further enhancements for screen reader users.
